docs/src/pages/guides/alerts.md

```diff
@@ -13,7 +13,7 @@ probes:
       - method: GET
         url: https://github.com
         alerts:
-          - query: response.size >= 10000
+          - assertion: response.size >= 10000
```

Comment:
Change the title of this section to Alert's Assertion

Reply:
docs was updated @nicnocquee 

<img width="1431" alt="Screen Shot 2022-09-14 at 13 32 48" src="https://user-images.githubusercontent.com/42835003/190077634-cb66b546-a06f-4dde-a3e2-f7ce61b73d69.png">

<img width="1369" alt="Screen Shot 2022-09-14 at 13 33 09" src="https://user-images.githubusercontent.com/42835003/190077660-0c6b62d2-d9f2-488c-856f-ca594ade1b6a.png">